app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -280,12 +286,20 @@ class MainActivity : AppCompatActivity() {
         // Create output options object which contains file + metadata
         val outputOptions = StorageUtils.getVideoMediaStoreOutputOptions(contentResolver)
 
+        // Play shutter sound
+        if (sharedPreferences.shutterSound) {
+            mediaActionSound.play(MediaActionSound.START_VIDEO_RECORDING)
```

Comment:
we need to somehow wait for this sound to stop playing before starting recording, any idea how?

Reply:
You can hear it?
It may be a blocking call?

Reply:
It's the opposite, it's completely async, so it's on the recording ;(

Reply:
Ugh